### DISCUSSION TITLE
Is there no HDR inactive function in the action shortcuts?

### DISCUSSION THREAD

**Discussion by duk6046** [Codectory/AutoActions#82] [Q&A] [answered]
v1.9.0 seems right-click HDR toggle menu is gone.

**Comment by Codectory** [comment]
Just create the necessary action in the settings.

duk6046 ***@***.***> schrieb am Fr. 31. Dez. 2021 um 10:22:

**Reply by duk6046** [reply]
active is possible, but impossible to inactive

**Comment by Codectory** [comment]
That’s not correct. Just leave the HDR Mode unchecked.
duk6046 ***@***.***> schrieb am Fr. 31. Dez. 2021 um 10:27:

**Reply by duk6046** [reply]
it seems not working

**Comment by Codectory** [comment]
Can you send me a screenshot of the action to disable it?

duk6046 ***@***.***> schrieb am Fr. 31. Dez. 2021 um 10:31:

**Reply by duk6046** [reply]
![image](https://user-images.githubusercontent.com/32978164/147815477-7c29399d-71bc-4586-bfc9-84359901a4f1.png)

**Reply by duk6046** [reply]
![image](https://user-images.githubusercontent.com/32978164/147815574-13adc542-4564-4ccf-bb85-ae1d37e0e8dd.png)

**Comment by Codectory** [comment, marked as answer]
Check “Set HDR” and uncheck “Auto HDR Mode”.
The is a wrong translation. Will fix this.

duk6046 ***@***.***> schrieb am Fr. 31. Dez. 2021 um 10:34:

**Reply by duk6046** [reply]
Oh it's working. thanks
